src/System.Private.CoreLib/shared/System/Double.cs

```diff
@@ -217,47 +246,47 @@ public unsafe override int GetHashCode()
         public override String ToString()
         {
             Contract.Ensures(Contract.Result<String>() != null);
-            return FormatProvider.FormatDouble(m_value, null, null);
+            return Number.FormatDouble(m_value, null, NumberFormatInfo.CurrentInfo);
```

Comment:
@jkotas, are you indicating, that for these deltas I should keep `Number.FormatDouble(m_value, null, NumberFormatInfo.CurrentInfo);` and manually port the `Number` implementation from https://github.com/dotnet/coreclr/blob/master/src/mscorlib/src/System/Number.cs to https://github.com/dotnet/corert/tree/master/src/System.Private.CoreLib/src/System (or shared/System)?

or are you indicating something else?